force-app/main/default/classes/UTIL_ACCT_Naming.cls

```diff
@@ -165,6 +165,12 @@ public class UTIL_ACCT_Naming {
             'OtherPostalCode, OtherCountry, OtherLatitude, OtherLongitude, ' +
             'Phone, Fax';
 
+        Set<String> objectFields = Schema.SObjectType.Contact.fields.getMap().keySet();
+        Boolean ispersonNameSuffixEnabled = objectFields.contains('suffix');
+        if (ispersonNameSuffixEnabled) {
+            dynamicSoql += ', Suffix ';
+        }
+
```

Comment:
Same here, not sure I understand this change in the scope of the test automation. Thanks

Reply:
[Reference](https://github.com/SalesforceFoundation/EDA/pull/1950#discussion_r798784099)

Reply:
as previously noted, I think we should ask our PM and other devs/leads about this, just to get more context about it and assess potential risks.